site/default/static/build.js

```diff
@@ -38,24 +45,12 @@ module.exports = function(options, folders){
 			if(options.debug) {
 				console.log("BUILD: Copying build to dist.");
 			}
-			return fsx.copy(path.join(folders.build, "dist"), path.join(folders.dist) );
-			// copy everything to DIST
-			/*return Q.all([
-				fsx.mkdirs( path.join(folders.dist,"bundles") ).then(function(){
-					return fsx.copy(path.join(folders.build,"bundles"), path.join(folders.dist,"bundles") );
-				}),
-				fsx.copyFrom(path.join( require.resolve("steal"), "..", "steal.production.js"), path.join(folders.dist,"steal.production.js") ),
-				fsx.copy( path.join(folders.build,"html5shiv.js"), path.join(folders.dist,"html5shiv.js")),
-
-				copyDir("fonts"),
-
-				copyDir("img"),
-				copyDir("templates")
-			]);*/
 
+			var promise = Q.all([
+				fsx.copy(path.join(folders.build, "dist"), path.join(folders.dist))
+			].concat(staticDistPromises));
```

Comment:
This code is duplicated here and in the `options.devBuild` statement branch; could this be refactored to not be repeated in both places?